webapp/src/components/form/FormMultiFile.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from "react";
+import FormGroup from "./FormGroup";
+import MultiFileComponent from './MultiFileComponent';
+import "./Style.css";
+
+
+class FormMultiFile extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            fileList: [{ name: null, file: null, delete: null, filePath: null }],
+            addError: false,
+        }
+    }
+
+    componentWillMount() {
+        if (this.props.value) {
+            this.setState({
+                fileList: this.props.value,
+            })
+        }
+    }
+
+
+    // add file
+    addFile = () => {
+        // variables
+        let condition = true;
+        let addError = false;
+        let handleList = this.state.fileList;
+        // check against empty fields
+        handleList.map(val => {
```

Comment:
No need to change it, just FYI, you can do this in a more readable way by using the `all` function:
`const condition = handleList.all(val=>val.file)`  and then `addError` is just `!condition`